examples/ziti-end-to-end/README.md

```diff
@@ -0,0 +1,172 @@
+
+Ziti Binaries and relevant configs.
+
+[OpenZiti](https://github.com/openziti/ziti/tree/main) contains downloads for the latest builds or you can build it yourself. There are many example configs [in the etc dir](https://github.com/openziti/ziti/tree/main/etc).
+
+I have set up helpful environment variable for how I chose to get setup. I cloned the Ziti repository for local builds and set `$ZITI_SOURCE` to that directory. I then went into that directory and ran 
+```
+go install ./...
+```
+
+This will output the binary to `$GOPATH/bin`. I then do the same thing with this repository to install the go-httpbin binaries
+
+## Starting up the Ziti Network:
+
+### Controller Setup
+
+First we run the ziti controller with the arguent being which ever config file we chose. For this example I am using the `ctrl.with.edge.yml` file bundled for examples in the ziti repository.
+```
+ziti-controller run --log-formatter pfxlog $ZITI_SOURCE/ziti/etc/ctrl.with.edge.yml
+```
+
+We then need to initialize the controller with an admin user. For ease of testing I just used `admin admin admin` for my short local test.
+
+```
+ziti agent controller init <username> <password> <name-of-user>
+```
+
+You can then login to the edge via `ziti edge login` and putting in your credentials.
+
+### Policies
+Next we create all of the relevant service policies:
+
+```
+ziti edge create service-policy dial-simple Dial \
+    --service-roles '#simple' \
+    --identity-roles '#simple-client'
+
+ziti edge create service-policy bind-simple Bind \
+    --service-roles '#simple' \
+    --identity-roles '#simple-server'
+```
+
+And the Edge router policies:
+
+```
+ziti edge create edge-router-policy simple-client \
+    --identity-roles '#simple-client' \
+    --edge-router-roles '#all'
+
+ziti edge create edge-router-policy simple-server \
+    --identity-roles '#simple-server' \
+    --edge-router-roles '#all'
+
+ziti edge create service-edge-router-policy simple \
+    --service-roles '#simple' \
+    --edge-router-roles '#all'
+```
+
+Now we create and enroll our client user
+
+```
+ziti edge create identity service simple-client \
+    --jwt-output-file simple-client.jwt \
+    --role-attributes simple-client,simple
+
+ziti edge enroll \
+    --jwt simple-client.jwt \
+    --out simple-client.json
+```
+
+And then create and enroll the server
+
+```
+ziti edge create identity service simple-server \
+    --jwt-output-file simple-server.jwt \
+    --role-attributes simple-server,simple
+
+ziti edge enroll \
+    --jwt simple-server.jwt \
+    --out simple-server.json
+```
+
+All that's left to do for this portion is to create the service
+```
+ziti edge create service echo --role-attributes simple
+```
+
+Now you can optionally delete the two jwt's we generated. We will not be using them in the rest of the example.
```

Comment:
On my computer the `enroll` command cleaned up the consumed JWT files automatically.